assembl/models/auth.py

```diff
@@ -759,6 +759,15 @@ def password_p(self):
     def password_p(self, password):
         from ..auth.password import hash_password
         if password:
+            # keep the current password to avoid the user to reuse it later
+            self.old_passwords.append(
```

Comment:
Do you not make any checks before appending?

Reply:
Which check do you want me to do? self.old_passwords is a relation, so we know it's a list.